scss/detail.scss

```diff
@@ -39,59 +47,87 @@
 .container {
   box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
   display: flex;
-  margin: 20px auto;
+  margin: 80px auto;
   padding: 20px;
   position: relative;
-  width: 650px;
+  width: 670px;
+  @media screen and (max-width: 1024px) {
```

Comment:
We use the media screen at the end of the file and target the classes inside of them